sycl/plugins/level_zero/usm_allocator.cpp

```diff
@@ -132,12 +131,6 @@ static class SetLimits {
     }
     MaxPoolableSize *= (1 << 20);
     MaxPoolSize *= (1 << 20);
-
-    if (Trace) {
-      std::cout << "MaxPoolableSize = " << MaxPoolableSize << std::endl;
-      std::cout << "Capacity        = " << Capacity << std::endl;
-      std::cout << "MaxPoolSize     = " << MaxPoolSize << std::endl;
-    }
   }
 } L;
```

Comment:
Is "L" some standard name for this, could you give it a more meaningful one?

Reply:
OK.